CHANGELOG.md

```diff
@@ -2,6 +2,42 @@
+* sample queries documentation link ([#570](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/issues/570)) ([5cfa5fa](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/commit/5cfa5faf85e00aadf837216c0d64e5acfc12ead4))
+* screen reader announces search results status message ([#644](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/issues/644)) ([012d794](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/commit/012d794e1513280d6a0a95fcd4f9c9235e2fa9cd))
+* screen reader support and focus indication on links ([#625](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/issues/625)) ([5268204](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/commit/5268204535de8cc7e682da9b023f7b443aa94eba))
+* select permissions pop up ([#559](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/issues/559)) ([36c1468](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/commit/36c146832108bb55b3d1c4027cab50499b765d4c))
+* Share query errors ([#501](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/issues/501)) ([13e244c](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/commit/13e244ca1e1a85238caead4ed205f37f09384685))
+* Show log-out on try it on mobile screens ([#569](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/issues/569)) ([18636e4](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/commit/18636e40b645171276cff4c3c1cd38f89d8d6464))
+* show template variables ([#287](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/issues/287)) ([65ba338](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/commit/65ba3388c7defa05b4effe7d93b111d15e3cdd70))
+* sidebar http methods ([#527](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/issues/527)) ([65ca584](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/commit/65ca5840db189f82927e68f5ba1d8ad25e40678e))
+* snippets generation for queries with search parameters ([#549](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/issues/549)) ([929f9fb](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/commit/929f9fba72ab6f307d675be711e9204a6748aa31))
+* Stop Infinite spinning when login fails ([#492](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/issues/492)) ([c490a94](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/commit/c490a941ba28eb5787f051384b289149d2fbb715))
+* try it logout ([#502](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/issues/502)) ([6ed2330](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/commit/6ed23300ab96c2e1b04400a81a823ea8a728a4ff))
+* validate json ([#577](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/issues/577)) ([26ddbb1](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/commit/26ddbb14439913ce511c66f0581a3f18d9181ea7))
+
 ### 4.1.4 (2020-07-21)
```

Comment:
should this be updated?

Reply:
It should. It's automated though. I can add it to tech debt to refine the automation by standard version